tests/stage_2/add-1.txt

```diff
@@ -1,2 +1,2 @@
-brun -s2 '(opt (com (q . (+ 6 55))))'
+run -s0 -s2 '(opt (com (q . (+ 6 55))))'
```

Comment:
This seems like a victim of automated search and replace.
```suggestion
run '(opt (com (q . (+ 6 55))))'
```

Reply:
yes..